fileops-service/values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for upload-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: fileops
+  tag: 252
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+
+appConfig:
+  port: 5078
+  env: "dev"
+  config_center_enabled: false
+  config_center_base_url: http://common.utility:5062/
+  namespace: ""
+
+  ROOT_PATH: "/data/core-storage"
+
+  LEVEL_DEFAULT: 30 # WARN
+  LEVEL_FILE: 30 # WARN
+  LEVEL_STDOUT: 30 # WARN
+  LEVEL_STDERR: 40 # ERROR
+
+  REDIS_HOST: "redis-master.redis"
+  REDIS_PORT: "6379"
+  REDIS_DB: "0"
+
+  DOWNLOAD_TOKEN_EXPIRE_AT: "86400"
+
+  OPEN_TELEMETRY_ENABLED: "True"
+  OPEN_TELEMETRY_HOST: "127.0.0.1"
+  OPEN_TELEMETRY_PORT: "6831"
+
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+container:
+  port: 5078
+
+initContainers: {}
+
+deploymentAnnotations: {}
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+
+resources: {}
```

Comment:
highly recommend setting resources in the defaults.

if you don't know/can't predict how much cpu&memory resources this service will have, i recommend setting higher values for now and then we will adjust later in the helmfile values.yaml:
```
resources:
  limits:
    cpu: 300m
    memory: 300Mi
  requests:
    cpu: 100m
    memory: 100Mi
    ```

Reply:
Different services need different resources. We don't know what resources this service will need yet, but that's okay: now we can configure them in the values file in the helmfile repo.